bms/templates/registration/create_bug.html

```diff
@@ -1,13 +1,243 @@
+                                <div class="col-sm-4">
+                                    <textarea class="form-control" cols="40" id="description" name="description" rows="3" placeholder="Enter Bug Description"></textarea>
+                                </div>
+                            </div>
+
+                            <div class="form-group required" id="nameofprodtype">
+                                <label for="bug_owner" class="control-label col-sm-2 col-sm-offset-2">Owner:</label>
+                                <div class="col-sm-4">
+									{{bug_form.bug_owner.errors}}
+													
+									{{ bug_form.bug_owner }}
+                                </div>
+                            </div>
+
+
+                            <div class="form-group" id="nameofprodtype">
+                                <label for="prodType" class="control-label col-sm-2 col-sm-offset-2">Assigned To:</label>
+                                <div class="col-sm-4">
+									{{bug_form.bug_assigned_to.errors}}
+													
+									{{ bug_form.bug_assigned_to }}
+                                </div>
+                            </div>
+
+                            <div class="form-group" id="nameoffile">
+                                <label for="bug_file" class="control-label col-sm-2 col-sm-offset-2">Upload File:</label>
+                                <div class="col-sm-4">
+                                    {{ bug_form.bug_file.errors }}
+                                   {{ bug_form.bug_file }}
+                                </div>
+                            </div>
+
+
+                            <div class="form-group" id="nameofbuton">
+                                <div class="controls">
+                                    <a class="btn btn-sm btn-primary col-sm-offset-6" type="button" href="/website_home/" id="cancel-button"> <strong>Cancel </strong> </a>
+                                    <button class="btn btn-sm btn-primary" type="submit"> <strong>Create</strong> </button>
+                                </div>
+                            </div>
+                        </form>
+                        {% endblock %}
+                    </div>
+                </div>
+            </div>
+
+        </div>
+    </div>
 </body>
-</html>
\ No newline at end of file
+<script type='text/javascript' src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script type='text/javascript' src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+<script type="text/javascript" src="{% static 'bms_app/js/style.js' %}"></script>
+<script type="text/javascript" src="{% static 'bms_app/js/create_prod.js' %}"></script>
+<script type="text/javascript" src="{% static 'bms_app/js/prod_list.js' %}"></script>
+
+</html>
```

Comment:
Header, navigation and footer sections should be in a common file, that should not repeat in very templates. Need to create new task in next sprint.